src/e2e/puppeteer/__tests__/color.ts

```diff
@@ -0,0 +1,35 @@
+import helpers from '../helpers'
```

Comment:
This looks good.

There are few other tests we need to port over from the old RTL test:

- Clear the text color when selecting white
- Clear background color when selecting text color
- Change color to black when setting background color (this is broken)